Automattic/wp-calypso
# DIFM: Manage Purchases: Show extra page price breakdown using price tiers

#### Proposed Changes

Adds a proper description and cost breakdown for DIFM purchase receipt view.

#### Testing Instructions
- Apply  D85709-code
- Apply D86680-code
- Sandbox Public API
- Use the below calypso live link go to `/start/do-it-for-me`
- Go through the flow and select more than 5 pages in the page picker step
- Complete purchase
- Go to purchases `http://calypso.localhost:3000/purchases/subscriptions/<site-slug>`
- Select the DIFM purchase and make sure you see below
<img width="895" alt="image" src="https://user-images.githubusercontent.com/3422709/184663490-66259458-2606-442e-bde5-d42ddefc90b9.png">

#### Pre-merge Checklist






Fixes 937-gh-Automattic/martech
